vendors/microchip/boards/curiosity_pic32mzef/aws_demos/config_files/aws_iot_network_config.h

```diff
@@ -39,8 +39,11 @@
  * each network types supported. Flags for all supported network types can be found
  * in "aws_iot_network.h"
  */
-
-#define configSUPPORTED_NETWORKS    ( AWSIOT_NETWORK_TYPE_WIFI )
+#ifdef PIC32_USE_ETHERNET
```

Comment:
Either one or the other is compiled in and we can't just switch at run time. That is why this is not OR'ed.